src/main/java/seedu/guest/logic/commands/EditCommand.java

```diff
@@ -100,8 +101,9 @@ private static Guest createEditedGuest(Guest guestToEdit, EditGuestDescriptor ed
         NumberOfGuests updatedNumberOfGuests = editGuestDescriptor.getNumberOfGuests()
                 .orElse(guestToEdit.getNumberOfGuests());
         IsRoomClean updatedIsRoomClean = editGuestDescriptor.getIsRoomClean().orElse(guestToEdit.getIsRoomClean());
+        Bill updatedBill = guestToEdit.getBill();
```

Comment:
Can I check why the editGuestDescriptor does not have a getBill() method?

Reply:
Since we will be implementing the `bill` command for editing the bill, the user will not be able to edit the bill through the edit command, nor instantiate the bill with a value using the add command. This design decision is intended to mirror the remark command tutorial.

Reply:
I see, thanks for explaining.